docs/BasicCommunication/GetSystemTime/index.md

```diff
@@ -0,0 +1,109 @@
+## GeSystemTime
```

Comment:
Typo

Reply:
@jacobkeeler, thank you
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/94/commits/d5cb22355a29c59cb210659f179ed201a455b5b3